Assets/MixedRealityToolkit/Providers/UnityInput/UnityTouchDeviceManager.cs

```diff
@@ -67,6 +69,19 @@ public override void Update()
             }
         }
 
+        private void StartTouchData(Touch touch,Ray ray)
+        {
+            UnityTouchController controller;
+
+            if (!ActiveTouches.TryGetValue(touch.fingerId, out controller))
+            {
+                return;
+            }
+
+            controller.StartTouch();
```

Comment:
this will cause a pointerdown and a holdstarted for every stationary touch.

Reply:
Sorry...
You're completely right.
This should call first time only.